package.json

```diff
@@ -75,30 +60,26 @@
     "jsdoc": "3.4.0",
     "jshint": "2.9.1",
     "json-loader": "0.5.4",
-    "load-grunt-config": "0.17.2",
-    "load-grunt-tasks": "3.2.0",
     "memdown": "1.0.0",
+    "mocha": "2.4.5",
```

Comment:
@zombieprocess, FYI, I _think_ I saw in a recent ansible script mocha get added globally.  IMHO, we can scrap that.  mocha really can be run from the projects that use it.  `nodeapi`, for instance, uses the `mocha` CLI. it can (and should) execute its local executable